__tests__/app.test.js

```diff
@@ -64,4 +64,18 @@ describe("GET /api/articles/:article_id", () => {
                 expect(body.msg).toBe("article not found")
             }) 
     })
+})
+
+describe.only("PATCH /api/articles/:article_id", () => {
+    test("responds with the updated article", () => {
```

Comment:
Try to remember to remove `.only` from anywhere in your tests before a commit, so that husky can make sure *all* your tests still pass.